### PR TITLE
added a test for isbn10 strings

### PR DESCRIPTION
**Pull request by jspaaks**
**Related issues**

Refs: #336, PR #335

**Describe the changes made in this pull request**

Added a test for ISBN10 strings

**Instructions to review the pull request**

**Review checklist**

- [x] Please check if all changes are recorded in `CHANGELOG.md` and adapt if necessary.
- [x] Please run tests locally. _Test should fail until we merge https://github.com/citation-file-format/citation-file-format/pull/335_

### CODE REVIEW

File: examples/1.2.0/pass/isbn10/CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cff-version: 1.2.0
+message: If you use this software in your work, please cite it using the following metadata
+title: "The Software"
+authors:
+  - name: "The Authors Team"
+references:
+  - authors:
+      -
+        family-names: Lass
```
Comment (jspaaks) [inline examples/1.2.0/pass/isbn10/CITATION.cff:9]:
```suggestion
      - family-names: Lass
```

**Comment by sdruskat** [comment]
Want me to take a look at this @jspaaks?

**Comment by jspaaks** [comment]
> Want me to take a look at this @jspaaks?

yes please

**Review by sdruskat — APPROVED** [review]
Thanks for adding this test!
Shouldn't the README contain something if it exists?

Good to go after doing something with the README (removal is also an option, but would prefer some minimal "docs" for this test example).

**Comment by jspaaks** [comment]
>  Shouldn't the README contain something if it exists?

Yes I forgot, added now

**Comment by sdruskat** [comment]
> > Shouldn't the README contain something if it exists?
> 
> Yes I forgot, added now

Good to go, thanks :taco:

**Pull request merged**